### PR TITLE
SED-3580-expose getLastExecutionByTaskId

### PR DESCRIPTION
**Pull request by lucianbaciu**

### CODE REVIEW

File: step-controller/step-controller-server/src/main/java/step/core/execution/ExecutionServices.java [hunk 1]
```diff
@@ -170,6 +167,21 @@ public Map<String, String> getExecutionsNamesByIds(List<String> ids) {
 		return executionAccessor.findByIds(ids).collect(Collectors.toMap(e -> e.getId().toHexString(), Execution::getDescription));
 	}
 
+	@Operation(description = "Returns the last execution triggered by a specific task.")
+	@POST
```
Comment (david-stephan) [inline step-controller/step-controller-server/src/main/java/step/core/execution/ExecutionServices.java:171]: @lucianbaciu I think  a GET method would be more appropriate in this case.

**Review by david-stephan — CHANGES_REQUESTED** [review]
Small change requested

**Review by david-stephan — APPROVED** [review]

**Comment by lucianbaciu** [comment]
@david-stephan please check out my last changes

**Review by david-stephan — CHANGES_REQUESTED** [review]
See individual comments

File: step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesService.java [hunk 2]
```diff
@@ -71,6 +73,21 @@ public TimeSeriesAPIResponse getTimeSeries(@NotNull FetchBucketsRequest request)
...
         }
     }
+
+    @Secured(right = "execution-read")
+    @POST
+    @Path("/report-nodes")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    public TimeSeriesAPIResponse getReportNodesTimeSeries(@NotNull FetchBucketsRequest request) {
+        // TODO permissions? we don't have projectId yet
```
[the remark this answers is not in the document]
Comment (lucianbaciu) [reply step-controller-plugins/step-controller-plugins-timeseries/src/main/java/step/plugins/timeseries/TimeSeriesService.java:83]: @david-stephan I'm sorry, this was an old comment before our conversations

**Review by david-stephan — DISMISSED** [review]

**Review by david-stephan — APPROVED** [review]

**Pull request merged**